x/incentives/keeper/distribute_test.go

```diff
@@ -1583,9 +1583,11 @@ func (s *KeeperTestSuite) TestAllocateAcrossGauges() {
 		defaultGroup      = deepCopyGroup(defaultGroup)
 		singleRecordGroup = deepCopyGroup(singleRecordGroup)
 
+		two = sdk.NewInt(2)
```

Comment:
why multiply everything by two here?

Reply:
Added comment:
```
// Double the volume configuration in poolmanager because we want the current volume to be
// updated relative to the existing values in gauge record state.
// The current volume is computed = poolmanager cumulative volume - gauge record cumulative volume.
```